libminifi/include/Exception.h

```diff
@@ -60,16 +60,17 @@ class Exception : public std::exception {
  public:
   // Constructor
   /*!
-   * Create a new flow record
+   * Create a new exception
    */
-  Exception(ExceptionType type, const char *errorMsg)
+  Exception(ExceptionType type, std::string errorMsg)
       : _type(type),
-        _errorMsg(errorMsg) {
+        _errorMsg(std::move(errorMsg)) {
   }
+
   // Destructor
-  virtual ~Exception() throw () {
+  virtual ~Exception() noexcept {
```

Comment:
I didn't notice that throw was here. Odd choice. The compiler usually generates an implicit noexcept unless told otherwise, so I'm tempted to do a little more testing of this across compilers. I'll do this before merge.

Reply:
throw() == noexcept, however throw() is deprecated since C++11.

Reply:
I'm not sure how that applies to my comment...but I think the end result is that I will still run basic tests across compilers. Are you suggesting we needn't run tests for verification and validation?